health_ping.py

```diff
@@ -57,10 +57,15 @@ def update_app_version(app_version, c_name, e_name):
       log.debug(f"First version entry = {version_key}:{version_data}")
       return
 
+    # Only add latest version to redis stream if it has changed since last entry.
     if last_version != app_version:
       redis.xadd(version_key, version_data, maxlen=200, approximate=False)
-      redis.json().set('latest:versions', f'$.{version_key}', version_key)
```

Comment:
This was the real issue, see `version_key` where it should have been `version_data`